Documentation/Index.rst

```diff
@@ -51,6 +51,3 @@ a GNU/GPL CMS/Framework available from http://typo3.org
 
    Introduction/Index
    Installation/Index
-   Configuration/Index
```

Comment:
Why do you remove these pages from the menu?

Reply:
When I used make docs then a warning appeared: [2024-10-15T14:08:28.055255+00:00] app.WARNING: Menu entry "Configuration/Index" was not found in the document tree. Ignoring it.
So I concluded that these entries can be deleted safely.

Reply:
The writer has not created an Index.rst file in folder Configuration, which was also not there. So I deleted this from the menu.

Reply:
Ok, seems like they never existed